### ISSUE TITLE
addConnectionMgmt.sql is incorrectly included in the core installation scripts (E)

### ISSUE THREAD

**Issue by smurthys** [DASSL/ClassDB#142]
The script `addConnectionMgmt.sql` is included in `prepareDB.psql` though the functionality `addConnectionMgmt.sql` adds is not essential to using ClassDB: the functionality added is useful, but not essential.

I propose excluding `addConnectionMgmt.sql` from `prepardDB.psql`.

**Comment by smurthys** [comment]
I also propose the following improvements :
- Rename `prepareDB.psql` to `prepareDBCore.sql`
- Create a `prepareDBAddOn.psql` with entries for scripts for each optional add-on feature.

Proposed organization for `prepareDBAddOn.psql`:
- Group script files by feature and dependency, with a blank line before and after each group, and a meaningful comment before the start of the group 
- As of now, most features need only one script file

**Comment by smurthys** [comment]
Also, we should classify each script file as "Core" or "AddOn" and add the file to appropriate `.psql` file

**Comment by smurthys** [comment]
We made the following the decision in the Skype call on Jan. 10, 2017 about this topic:
- Scripts are of three kinds: core, recommended, and optional
- Every script is of exactly one kind
- Each kind of script will be stored in its own kind-specific sub-directory (of directory `src`). The sub-directory names are: `core`, `reco`, and `opt`
- Each script's filename has a suffix to denote its kind. Though they may seem redundant, the suffixes make it easier to spot and discuss files. The suffixes are `Core`, `Reco`, and `Opt`. For example, `addRoleBaseMgmtCore.sql`.

In addition to the structure summarized above, I propose server-level scripts and db-level scripts be stored in separate sub-directories, and each of them further containing kind-specific sub-directories. A kind-specific sub-directory may be not created at all if no script of that kind is present.

The following is the proposed directory structure for source files:

```
src
  -- server
    -- core
    -- reco
    -- opt
  -- db
    -- core
    -- reco
    -- opt
```

**Comment by afig** [comment]
Resolved by #164
